### DISCUSSION TITLE
Download audio

### DISCUSSION THREAD

**Discussion by Musky80** [tughi/aggregator-android#47] [Ideas]
I like this application, but I also use RSS readers to subscribe to podcasts, and in this application I do not see the audio file that is part of the post at all. The link is in the enclosure tag. Does it add up?

**Comment by tughi** [comment]
I don't follow podcasts myself... therefore I never implemented such a feature.

This app is based on more or less my needs. But I would gladly accept a PR that brings a clean implementation for this.

**Comment by SmutnyStefan** [comment]
Please, implement that. It is the only major feature that is imho missing here

**Comment by tughi** [comment]
Do you have examples of such feeds?
